spec/calendar.html

```diff
@@ -1505,12 +1505,8 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
       <emu-alg>
         1. Let _calendar_ be the *this* value.
         1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
-        1. Set _fields_ to ? ToObject(_fields_).
```

Comment:
Why is it `ToObject` and not "If fields is not an Object, throw a TypeError" ?

And was it an accidentally normative change to remove it here?